examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -58,14 +56,18 @@ public ArmeriaClientConfigurator armeriaClientConfigurator() {
             final CircuitBreakerStrategy strategy = CircuitBreakerStrategy.onServerErrorStatus();
             builder.decorator(new CircuitBreakerHttpClientBuilder(strategy).newDecorator());
 
-            // Automatically retry a request when the server returns a 5xx response.
-            builder.decorator(RetryingHttpClient.newDecorator(RetryStrategy.onServerErrorStatus()));
-
-            // Use a custom client factory in order to disable the certificate validation,
-            // which means any certificate received from the server will be accepted.
-            final ClientFactory clientFactory = new ClientFactoryBuilder().sslContextCustomizer(
-                    b -> b.trustManager(InsecureTrustManagerFactory.INSTANCE)).build();
-            builder.factory(clientFactory);
+            // Set a custom client factory.
+            builder.factory(clientFactory());
```

Comment:
Probably better letting Spring inject the `ClientFactory` as a parameter of `armeriaClientConfigurator()` method?